runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigureTask.java

```diff
@@ -162,11 +162,14 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
 
         if (config.configSyntaxMustache())
         {
-            configText = Mustache.resolve(configText, System::getenv);
+            ExpressionResolver expressionResolver = ExpressionResolver.instantiate();
+            for (String context: expressionResolver.names())
+            {
+                configText = expressionResolver.resolve(context, configText);
+            }
```

Comment:
We can do this in a single pass instead of a loop, no need to pass each context separately.